WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginPrologueFragment.java

```diff
@@ -48,6 +49,24 @@ public void onPageScrolled(int position, float positionOffset, int positionOffse
 
             @Override
             public void onPageSelected(int position) {
+                switch (mPagerAdapter.getTag(position)) {
```

Comment:
Hey @theck13 , what do you think about making `getTag()` a static method (and the `mTags` field static as well)? The `mPagerAdapter` reference won't be needed in that case.

Reply:
That works for me.  I updated the `getTag()` method and `mTags` array in bc44e1941a3c485d407dc28230ee22f301e4a052.